pool/pool_aio_test.go

```diff
@@ -108,14 +131,85 @@ func testEaclTable(containerID cid.ID) eacl.Table {
 	return table
 }
 
-func TestPoolInterfaceWithAIO(t *testing.T) {
+func TestPoolAio(t *testing.T) {
+	for _, version := range versions {
+		image := fmt.Sprintf("%s:%s", version.image, version.version)
+
+		t.Run(image, func(t *testing.T) {
+			t.Parallel()
+
+			ctx, cancel := context.WithCancel(context.Background())
+			aioContainer := createDockerContainer(ctx, t, image)
+
+			// for instance: localhost:32781
+			nodeEndpoint, err := aioContainer.Endpoint(ctx, "")
+			require.NoError(t, err)
+
+			runTests(ctx, t, nodeEndpoint)
+
+			err = aioContainer.Terminate(ctx)
+			require.NoError(t, err)
+			cancel()
+			<-ctx.Done()
+		})
+	}
+}
+
+func runTests(_ context.Context, t *testing.T, nodeEndpoint string) {
+	nodeAddr := nodeAddress(nodeEndpoint)
+
+	t.Run("PoolInterfaceWithAIO", func(t *testing.T) {
+		testPoolInterfaceWithAIO(t, nodeAddr)
+	})
+
+	t.Run("PoolWaiterWithAIO", func(t *testing.T) {
+		testPoolWaiterWithAIO(t, nodeAddr)
+	})
+
+	t.Run("ClientWaiterWithAIO", func(t *testing.T) {
+		testClientWaiterWithAIO(t, nodeAddr)
+	})
+}
+
+func createDockerContainer(ctx context.Context, t *testing.T, image string) testcontainers.Container {
+	req := testcontainers.ContainerRequest{
+		Image:        image,
+		WaitingFor:   wait.NewLogStrategy("Serving neofs rest gw").WithStartupTimeout(45 * time.Second),
+		Name:         "sdk-poll-tests-" + strconv.FormatInt(time.Now().UnixNano(), 36),
+		Hostname:     "aioa",
+		ExposedPorts: []string{"8080/tcp"},
+		Env: map[string]string{
+			"REST_GW_WALLET_PATH":       "/config/wallet-rest.json",
+			"REST_GW_WALLET_PASSPHRASE": "one",
+			"REST_GW_WALLET_ADDRESS":    "NPFCqWHfi9ixCJRu7DABRbVfXRbkSEr9Vo",
+			"REST_GW_PEERS_0_ADDRESS":   "localhost:8080",
+			"REST_GW_LISTEN_ADDRESS":    "0.0.0.0:8090",
+		},
+	}
+	aioC, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: req,
+		Started:          true,
+	})
+	require.NoError(t, err)
+
+	// Have to wait this time. Required for new tick event processing.
+	// Should be removed after fix epochs in AIO start.
```

Comment:
do we have an issue for that?

Reply:
We [had](https://github.com/nspcc-dev/neofs-aio/issues/54), we thought it was fixed, but looks like the problem still exists

Reply:
i mean, it the issue is fixed, will that line be removed? or everybody will forget about it?

Reply:
That issue fixed, but not all problems were solved. I will create another issue for that. Thus these rows are actual

Reply:
> Thus these rows are actual

yes, that is good. how can we be sure that lines (in _that_ repo) will be removed after fixing `aio` problems?

Reply:
> the second one for updating this repo after that

that is what bothers me, yes

Reply:
AIO issue https://github.com/nspcc-dev/neofs-aio/issues/59. And issues to fix after:
* https://github.com/nspcc-dev/neofs-sdk-go/issues/476
* https://github.com/nspcc-dev/neofs-rest-gw/issues/94